bluebrain/repo-bluebrain/packages/nmodl/package.py

```diff
@@ -56,6 +59,10 @@ class Nmodl(CMakePackage):
     depends_on("spdlog")
     depends_on("py-find-libpython", type=("run",))
 
+    depends_on("py-pytest@3.3.0:", type="test", when="+testing")
```

Comment:
I'm also wondering - is the `test` type handled properly now? IIRC, Gitlab uses `build-env` to run `ctest` :grimacing:

Reply:
It does use that. Would a `("build", "run",)` type be better?

Reply:
No `run`, that brought us to this situation initially. `build` + `test` maybe, I'm still looking.